pages/enterprise/cloud_forwarder.rst

```diff
@@ -0,0 +1,420 @@
+Introduction
+************
+The Graylog Cloud Forwarder is the solution to get your data into Graylog Cloud. Graylog Cloud allows you to set up
+a connection so one or more Forwarders can send messages. Since it’s hosted in the cloud you can run the Forwarder 
+anywhere. In addition, your data has improved uptime and reliability.
+
+.. image:: /images/forwarder_diagram.png
+
+The Cloud Forwarder is not a one-and-done migration tool. In fact, it’s an integral part of your cloud architecture. 
+Your Forwarder must run 24-7 in your environment in order to get data in the cloud.
+
+********
+Security
+********
+
+The Forwarder connects to Graylog Cloud over TLS. This ensures data moves securely and reliably. 
+
+Another layer of security required for the Forwarder is an API token. To ensure only Forwarders you trust can connect
+to your Graylog Cloud instance, Forwarders must be assigned and configured with a token to authenticate with Graylog 
+Cloud. Any tokens used by the Forwarder must belong to the user Forwarder System User (built-in). You can find 
+information about creating an `API token in the documentation <https://docs.graylog.org/en/4.0/pages/configuration/rest_api.html?highlight=token#creating-and-using-access-token>`__.
+
+*************
+Input Profile
+*************
+
+Input Profiles are a set of Inputs that can be configured to run on one or more Forwarders of your choice. 
+This aspect is critical to your Forwarder because it provides a gateway to move data to Graylog Cloud.
+
+Input Profiles helps you to avoid re-doing the same configuration for all Forwarders since you can start new Forwarders 
+and assign them a set of Inputs you already defined and tested beforehand. This is especially helpful if you want to collect the same kind of logs in different parts of your Infrastructure or to have a more redundant set-up.
+
+As you create your Cloud Forwarder, you must create an Input Profile from the wizard, explained in Add Inputs. 
+Provide a descriptive name for the Input Profile, a short description of its intention, and then create all the input 
+necessary for the Forwarder.
+
+*************
+Configuration
+*************
+
+File
+====
+
+On the client-side of your Forwarder, modify the configuration file to address your Graylog host and authentication. 
+The default values for ``forwarder.conf`` are:
+
+* ``forwarder_server_hostname``: (required) The Graylog Cloud Forwarder ingest hostname (eg. ``ingest-<your-account>.graylog.cloud``). It’s found on the Forwarder Setup Wizard in Graylog Cloud.
+* ``forwarder_grpc_api_token``: (required) The API Token for authenticating the Forwarder.
+
+Here’s an example of the configuration file, with the following values:
+
+.. code-block:: bash
+
+    $ forwarder_server_hostname = ingest-<your-account-url>.graylog.cloud
+    $ forwarder_grpc_api_token = <your-api-token>
+
+Wizard
+======
+
+When you login to the Cloud instance, you’ll walk through a guided process to configure your Forwarder.
```

Comment:
```suggestion
When you login to your Cloud instance, you can use a guided process to configure your Forwarder.
```

This seems to hint that the Forwarder setup will be immediately presented after login. What do you think about this adjustment?